scripts/main.js

```diff
@@ -73,6 +74,15 @@ module.exports = (robot) => {
             res.reply("Milestones successfully closed")
         }
     })
+    
+    robot.hear(/bot issue "([^"]+)"(?: (?:about|regarding|re|body|description) "([^"]+)")?(?: in "([\w\d-_]+)(?:\/([\w\d-_]+))?")/i, (res) => {
```

Comment:
So I also can write: "bot issue "My issue" re "This is test issue" in "datahq/frontend" ?

Reply:
so I replaced `about` with `re`

Reply:
@anuveyatsu yes, any of the following words: about, regarding, re, body, description. But I think I will remove the rest. What do you think?

Reply:
@Mikanebu I think it's better to keep

Reply:
@anuveyatsu Ok, I will create an issue about lint